gobblin-runtime/src/test/java/org/apache/gobblin/runtime/spec_store/MysqlSpecStoreTest.java

```diff
@@ -66,16 +68,23 @@ public void setUp() throws Exception {
         .addPrimitive(ConfigurationKeys.STATE_STORE_DB_TABLE_KEY, TABLE)
         .build();
 
-    this.specStore = new MysqlSpecStore(config);
+    this.specStore = new MysqlSpecStore(config, new TestSpecSerDe());
   }
 
   @Test
-  public void testAddGetSpec() throws Exception {
+  public void testAddSpec() throws Exception {
     this.specStore.addSpec(this.flowSpec1);
     this.specStore.addSpec(this.flowSpec2);
 
     Assert.assertTrue(this.specStore.exists(this.uri1));
+    Assert.assertTrue(this.specStore.exists(this.uri2));
     Assert.assertFalse(this.specStore.exists(URI.create("dummy")));
+  }
+
+  @Test
+  public void testGetSpec() throws Exception {
```

Comment:
Can we add a test for getSpecs() where the de-serialization of a spec fails, but getSpecs() returns other valid specs?